### PR TITLE
Login notifications

### PR DESCRIPTION
**Pull request by a0eoc**
Wording can be improved here.

### CODE REVIEW

File: handlers/user/login.go [hunk 1]
```diff
@@ -122,5 +122,30 @@ func LoginPost(c *fiber.Ctx) error {
...
+				"<p>We noticed a new login to your UserStyles.world account.</p>\n" +
+				"<br>" +
+				"<b>If that was you, you can safely ignore this e-mail.\n" +
+				"<br>" +
+				"<b>If that wasn't you, <a target=\"_blank\" clicktracking=\"off\" href=\"https://userstyles.world/account#password\">change your password</a>.\n").
+			SetContentType("text/html")
+
+		utils.NewEmail().
+			SetTo(user.Email).
+			SetSubject("New login").
```
Comment (vednoc) [inline handlers/user/login.go:144]:
```suggestion
			SetSubject("Sign in notice").
```

File: handlers/user/login.go [hunk 2]
```diff
@@ -122,5 +122,30 @@ func LoginPost(c *fiber.Ctx) error {
...
+				"If that wasn't you, change your password:\n" +
+				"https://userstyles.world/account#password")
+		partHTML := utils.NewPart().
+			SetBody("<p>Hi " + user.Username + ",</p>\n" +
+				"<br>" +
+				"<p>We noticed a new login to your UserStyles.world account.</p>\n" +
+				"<br>" +
+				"<b>If that was you, you can safely ignore this e-mail.\n" +
+				"<br>" +
+				"<b>If that wasn't you, <a target=\"_blank\" clicktracking=\"off\" href=\"https://userstyles.world/account#password\">change your password</a>.\n").
```
Comment (vednoc) [inline handlers/user/login.go]:
```suggestion
				"We noticed a new sign in to your UserStyles.world account.\n\n" +
				"If that was you, you can safely ignore this email.\n" +
				"If that wasn't you, please change your password:\n" +
				"https://userstyles.world/account#password")
		partHTML := utils.NewPart().
			SetBody("<p>Hi " + user.Username + ",</p>\n" +
				"<br>\n" +
				"<p>We noticed a new sign in to your UserStyles.world account.</p>\n" +
				"<br>\n" +
				"<b>If that was you, you can safely ignore this email.\n" +
				"<br>\n" +
				"<b>If that wasn't you, <a target=\"_blank\" clicktracking=\"off\" href=\"https://userstyles.world/account#password\">please change your password</a>.\n").
```

**Review by vednoc — CHANGES_REQUESTED** [review]

File: handlers/user/login.go (hunk 1 above)
Comment (a0eoc) [reply handlers/user/login.go:144]: I searched for my login emails to write that. Never seen `Sign in notice`. I think `New log-in` would be better.

File: handlers/user/login.go (hunk 2 above)
Comment (a0eoc) [reply handlers/user/login.go]: I already updated that file to add `\n`s. Will commit manually.

**Review by vednoc — APPROVED** [review]

**Pull request merged**